src/app.js

```diff
@@ -1 +1,48 @@
-// write code here
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const path = require('path');
+
+function moveFile() {
+  const params = process.argv.slice(2);
+  const source = params[0];
+  const destination = params[1];
+
+  if (!source || !destination) {
+    console.error('Please provide a source file and a destination file.');
+
+    return;
+  }
+
+  if (!fs.existsSync(source)) {
+    console.error('File does not exist.');
+
+    return;
+  }
+
+  const isExistDirectory = fs.existsSync(destination);
```

Comment:
The variable `isExistDirectory` is misleadingly named. It checks if the destination exists, not specifically if it's a directory. Consider renaming it to `destinationExists`.